main.go

```diff
@@ -1,144 +1,18 @@
 package main
 
 import (
-	"errors"
 	"fmt"
-	"io/ioutil"
 	"log"
 	"os"
-	"strings"
 
-	"github.com/hashicorp/vault/api"
-	"github.com/mitchellh/go-homedir"
+	homedir "github.com/mitchellh/go-homedir"
```

Comment:
I'm guessing your editor did this for you automagically? I'm really not a big fan of this auto-rewrite, but that's just me.